posquit0/Awesome-CV
# Fix issue where copying bullets from the resulting pdf results in text with no whitespace and add `cvsubitems`

This solution is based on @chelseanbr's comment from July 28, 2023, https://github.com/posquit0/Awesome-CV/issues/71#issuecomment-1654964285.

Additionally this PR creates a new environment `cvsubitems`, that is functionally the same as `cvitems`, with the same workaround to allow reliable copypasting and parsing by ATS systems, but without some adjustments to vertical space that only make sense for `cvitems`

This fixes #137

## Test Plan:

1. run `make`
2. Open all resulting PDFs and copy text from bullet points into a text editor, review that the copied text has spaces as expected.
3. Add a `cvsubitems` environment to the first `cventry` in experience to test that this also works as expected
4. Do the same copypaste test on `cv.pdf`, guaranteeing that we copy the subitems text.
